src/reference/03-Developers-Guide/10-DevGuide-Notes/00.md

```diff
@@ -0,0 +1,8 @@
+---
+out: DevGuide-Notes.html
+---
+
+Notes
```

Comment:
This was really about the architectural core of sbt.  I would like to keep it around/live.